[docs] Updated documentation regarding external elements (#602)
shipshapecode/ember-flatpickr: tests/dummy/app/templates/docs/usage.md

The wrap option for flatpickr causes flatpickr to search its child elements for elements annotated with certain attributes. With ember-flatpickr this can be accomplished with the following

```handlebars

> <!-- components/my-date-picker.hbs -->

Reviewer: ```suggestion
{{!-- components/my-date-picker.hbs --}}
```

```

```javascript
// components/my-date-picker.js

import Component from '@glimmer/component'
import { tracked } from '@glimmer/tracking'
import { action } from '@ember/object'

export default MyDatePicker extends Component {

  @tracked flatpickrRef = null

  @action
  onReady(_selectedDates, _dateStr, flatpickrRef) {
    this.flatpickrRef = flatpickrRef
  }

  @action
  toggleCalendar(e) {
    e.stopPropagation()
    this.flatpickrRef.toggle();
  }

  @action
  clearCalendar(e) {
    e.stopPropagation()
    this.flatpickrRef.clear();
  }

}
```

**Note:** In the above example, the `toggleCalendar` and `clearCalendar` actions

> receive an argument `e` (the click event) and call `e.stopPropagation` on said event. FlatPickr

Reviewer: ```suggestion
receive an argument `e` (the click event) and call `e.stopPropagation` on said event. flatpickr
```

> is designed to immediately close the calendar popup when user's click outside of it's bounding DOM element. Clicking the above _toggle_ button will call the "open" method on the FlatPickr instance, but since our button lives outside of the bounds of the pickr's calendar, FlatPickr immediately closes the calendar popup. By stopping the propagation of our external click event, FlatPickr does not detect the outside click and the calendar opens as expected.

Reviewer: ```suggestion
is designed to immediately close the calendar popup when user clicks outside of its bounding DOM element. Clicking the above _toggle_ button will call the "open" method on the flatpickr instance, but since our button lives outside of the bounds of the picker's calendar, flatpickr immediately closes the calendar popup. By stopping the propagation of our external click event, flatpickr does not detect the outside click and the calendar opens as expected.
```
